### PR TITLE
Add landing page message #1 - Done

### PR DESCRIPTION
**Pull request by carti**
I had completed the task

### CODE REVIEW

**Review by Karthi0505 — CHANGES_REQUESTED** [review]
Could you remove the word "Bootstrap" from <h2>

File: index.html [hunk 1]
```diff
@@ -10,7 +10,9 @@
 <body>
 
 <div class="container-fluid p-5 bg-primary text-white text-center">
-  <h1>My First Bootstrap Page</h1>
+  <h1>Adding a Landing page message by Carti</h1>
+  
+  <h2>My First Bootstrap Page</h2>
```
Comment (Karthi0505) [inline index.html]: Could you remove the word "Bootstrap" from

Comment (Karthi0505) [inline index.html]: Add a para

**Review by Karthi0505 — CHANGES_REQUESTED** [review]
Pls complete 2 changes

File: index.html (hunk 1 above)
Comment (carti) [reply index.html]: Sure I will remove it

Comment (carti) [reply index.html]: I'll add a para

**Review by Karthi0505 — APPROVED** [review]

**Pull request merged**